cmd/openshift-install/agent.go

```diff
@@ -50,7 +50,7 @@ var (
 
 				timer.StartTimer("Agent Create Image Complete")
 
-				err := agentcmd.BuildImage()
+				err := agentcmd.BuildImage(rootOpts.dir)
```

Comment:
This will conflict with #5906. It would be better if we could build on top of that.